README.md

```diff
@@ -83,9 +84,35 @@ hover text.  For other available options, see the
 
 Learn more about using KaTeX [on the website](https://katex.org)!
 
-## Contributing
+## Contributors
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="https://github.com/KaTeX/KaTeX/graphs/contributors"><img src="https://opencollective.com/katex/contributors.svg?width=890&button=false" /></a>
```

Comment:
Is this plot supposed to do something? I don't see anything (at least in GitHub dark mode).

Reply:
It should show contributors like this, but due to https://github.com/opencollective/opencollective/issues/4517 it doesn't work now:

![](https://opencollective.com/jest/contributors.svg?width=890&button=false)

Reply:
Reading some new posts on that thread, it looks like https://contributors-svg.opencollective.com/katex/contributors.svg?width=890&button=false works:

![](https://contributors-svg.opencollective.com/katex/contributors.svg?width=890&button=false)

Reply:
```suggestion
<a href="https://github.com/KaTeX/KaTeX/graphs/contributors"><img src="https://contributors-svg.opencollective.com/katex/contributors.svg?width=890&button=false" /></a>
```